tests/Application/templates/bundles/SyliusShopBundle/layout.html.twig

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+
+<html lang="{{ app.request.locale|slice(0, 2) }}">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <title>{% block title %}{{ sylius.channel.name }}{% endblock %}</title>
+
+    <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
+
+    {% block metatags %}
+    {% endblock %}
+
+    {% block stylesheets %}
+        {{ sylius_template_event('sylius.shop.layout.stylesheets') }}
+    {% endblock %}
+
+    {{ sylius_template_event('sylius.shop.layout.head') }}
+</head>
+
+<body class="pushable">
+{{ sylius_template_event('sylius.shop.layout.before_body') }}
+<div class="pusher">
+    {% block top %}
+        <div id="menu" class="ui large sticky inverted stackable menu">
+            {{ sylius_template_event('sylius.shop.layout.topbar') }}
+        </div>
+    {% endblock %}
+
+    <div class="ui container">
+        {% if ssgo_variant('black_friday_test', 'original') %}
```

Comment:
Kunne man forestille sig at man kunne lave en "experiment context" som kan bruges, så man er fri for at gentage eksperiment navnet?

Altså noget ala
```
class ExperimentContext {
  __contruct(Experiment $experiment) { ... }

  __invoke($variant): bool { ... }
```

og så kunne man i f.eks. layout.html.twig gøre noget ala:
```
{% set black_friday = ssgo_experiment('black_friday_test') %}

{% if black_friday('original') %}
...
{% else if black_friday('remove_header') %}
...
{% endif %}
```

Reply:
Det kunne sagtens være. Det ser jeg lige på :)

Reply:
Det ville også gøre `black_friday` tilgængelig i alle child templates. Om man så synes det er god skik eller ej, at gøre brug af,  ved jeg ikke :)

Reply:
Kan ikke lige finde en decideret smuk løsning synes jeg, så jeg merger den her PR, og tilføjer det her som et issue